tools/code-analyzer/src/lib/__tests__/fixtures/diff.txt

```diff
@@ -965,7 +965,7 @@ index 99b4cbaff3..b8304993ff 100644
 @@ -20,7 +20,8 @@
  		"git:update-hooks": "rm -r .git/hooks && mkdir -p .git/hooks && husky install",
  		"storybook": "./tools/storybook/import-wp-css-storybook.sh && BABEL_ENV=storybook STORYBOOK=true start-storybook -c ./tools/storybook/.storybook -p 6007 --ci",
- 		"storybook-rtl": "USE_RTL_STYLE=true pnpm run storybook",
+ 		"storybook-rtl": "USE_RTL_STYLE=true pnpm storybook",
```

Comment:
Hmm, find and replace probably didn't need to touch this fixture file :D 

but also... I don't *think* it matters..

Reply:
It doesn't!